[36mATRI[0m | 03-09 16:01:45 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7fa792daa040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7fa789e00f10>
         │    └ <function MultiCommand.invoke at 0x7fa792dae0d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7fa79af706a0>
           │               │       │       └ <function Command.invoke at 0x7fa792daaaf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7fa79af706a0>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7fa789cd59d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7fa79af706a0>
           │   │      │    └ <function run at 0x7fa789dff8b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7fa792da68b0>
           └ <click.core.Context object at 0x7fa79af706a0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7fa789def670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7fa79af5d940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7fa79b08c430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7fa790c6e790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7fa790c68310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7fa7895e56a0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7fa790c681f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7fa7895e56a0>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7fa790c68280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7fa7895e56a0>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7fa79af5d940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7fa79b08c430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-09 16:06:17 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f86dd091040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f86dd9174f0>
         │    └ <function MultiCommand.invoke at 0x7f86dd0950d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f86e5204370>
           │               │       │       └ <function Command.invoke at 0x7f86dd091af0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f86e5204370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f86d3fbc9d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f86e5204370>
           │   │      │    └ <function run at 0x7f86d40e78b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f86dd08d8b0>
           └ <click.core.Context object at 0x7f86e5204370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f86d40d6670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7f86e5244940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7f86e5373430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7f86daf55790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7f86daf4f310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7f86d38ca9a0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7f86daf4f1f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7f86d38ca9a0>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7f86daf4f280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7f86d38ca9a0>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7f86e5244940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7f86e5373430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-09 16:19:01 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7fb151ddc040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7fb1526624f0>
         │    └ <function MultiCommand.invoke at 0x7fb151de00d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7fb159f4f370>
           │               │       │       └ <function Command.invoke at 0x7fb151ddcaf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7fb159f4f370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7fb148d079d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7fb159f4f370>
           │   │      │    └ <function run at 0x7fb148e328b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7fb151dd88b0>
           └ <click.core.Context object at 0x7fb159f4f370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7fb148e21670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7fb159f8f940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7fb15a0be430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7fb14fca0790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7fb14fc9a310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7fb148615970>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7fb14fc9a1f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7fb148615970>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7fb14fc9a280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7fb148615970>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7fb159f8f940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7fb15a0be430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-09 18:34:21 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7fb151ddc040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7fb1526624f0>
         │    └ <function MultiCommand.invoke at 0x7fb151de00d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7fb159f4f370>
           │               │       │       └ <function Command.invoke at 0x7fb151ddcaf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7fb159f4f370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7fb148d079d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7fb159f4f370>
           │   │      │    └ <function run at 0x7fb148e328b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7fb151dd88b0>
           └ <click.core.Context object at 0x7fb159f4f370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7fb148e21670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7fb14af0b940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7fb14af0b4c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7fb148c2e670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7fb148c2ea60>
    └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7fb148c2eaf0>
           │       │   └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
           │       └ <function run at 0x7fb14e663430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7fb0dd5402c0>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-265255' coro=<_command() done, defined at /usr/local/python...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0dc164b50>
          │            │        │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '!stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │            │        │    └ GroupMessageEvent(time=1646822056, self_id=507346814, post_type='message', sub_type='normal', user_id=2095652005, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7fb148d49670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-265255' coro=<_command() done, defined at /usr/local/python...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0dc164b50>
          │       │   │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '!stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │       │   │    └ GroupMessageEvent(time=1646822056, self_id=507346814, post_type='message', sub_type='normal', user_id=2095652005, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │       └ <function Matcher.run at 0x7fb14af48b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-265255' coro=<_command() done, defined at /usr/local/python...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0dc164b50>
          │    │          │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '!stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │    │          │    └ GroupMessageEvent(time=1646822056, self_id=507346814, post_type='message', sub_type='normal', user_id=2095652005, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │    └ <function Matcher.simple_run at 0x7fb14af48af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'bot': <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>, 'event': GroupMessageEvent(time=1646822056, self_id=5...
                 │    └ <function _ at 0x7fb12f669dc0>
                 └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 490, in _
    await stock.finish(
          │     └ <classmethod object at 0x7fb14af40a90>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 464, in finish
    await cls.send(message, **kwargs)
          │   │    │          └ {'at_sender': True}
          │   │    └ '\n🧾 Successfully sold 6 shares earning 1858.62 coins.\nYou have 25 shares left now.\n🟩 Current stock: 309.77 coins,  8.74%'
          │   └ <classmethod object at 0x7fb14af40a30>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 449, in send
    return await bot.send(event=event, message=_message, **kwargs)
                 │   │          │              │           └ {'at_sender': True}
                 │   │          │              └ '\n🧾 Successfully sold 6 shares earning 1858.62 coins.\nYou have 25 shares left now.\n🟩 Current stock: 309.77 coins,  8.74%'
                 │   │          └ GroupMessageEvent(time=1646822056, self_id=507346814, post_type='message', sub_type='normal', user_id=2095652005, message_typ...
                 │   └ <function Bot.send at 0x7fb148d49b80>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 247, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
                 │    │                      │     │      │          └ {'at_sender': True}
                 │    │                      │     │      └ '\n🧾 Successfully sold 6 shares earning 1858.62 coins.\nYou have 25 shares left now.\n🟩 Current stock: 309.77 coins,  8.74%'
                 │    │                      │     └ GroupMessageEvent(time=1646822056, self_id=507346814, post_type='message', sub_type='normal', user_id=2095652005, message_typ...
                 │    │                      └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                 │    └ <attribute '__class__' of 'object' objects>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 175, in send
    return await bot.send_msg(**params)
                 │              └ {'user_id': 2095652005, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '20...
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 208, in call_api
    return await super().call_api(api, **data)
                                  │      └ {'user_id': 2095652005, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '20...
                                  └ 'send_msg'
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 117, in call_api
    raise exception
          └ <ActionFailed data=None, echo={'seq': 35}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 94, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'user_id': 2095652005, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '20...
                   │    │       │         │     └ 'send_msg'
                   │    │       │         └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                   │    │       └ <function Adapter._call_api at 0x7fb148d49f70>
                   │    └ <nonebot.adapters.onebot.v11.adapter.Adapter object at 0x7fb148e378b0>
                   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/adapter.py", line 107, in _call_api
    return _handle_api_result(
           └ <function _handle_api_result at 0x7fb148ddb5e0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/utils.py", line 81, in _handle_api_result
    raise ActionFailed(**result)
          │              └ {'data': None, 'echo': {'seq': 35}, 'msg': 'SEND_MSG_API_ERROR', 'retcode': 100, 'status': 'failed', 'wording': '请参考 go-cqhtt...
          └ <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>

nonebot.adapters.onebot.v11.exception.ActionFailed: <ActionFailed data=None, echo={'seq': 35}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
[36mATRI[0m | 03-09 18:34:46 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7fb151ddc040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7fb1526624f0>
         │    └ <function MultiCommand.invoke at 0x7fb151de00d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7fb159f4f370>
           │               │       │       └ <function Command.invoke at 0x7fb151ddcaf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7fb159f4f370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7fb148d079d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7fb159f4f370>
           │   │      │    └ <function run at 0x7fb148e328b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7fb151dd88b0>
           └ <click.core.Context object at 0x7fb159f4f370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7fb148e21670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7fb14af0b940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7fb14af0b4c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7fb148c2e670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7fb148c2ea60>
    └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7fb148c2eaf0>
           │       │   └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
           │       └ <function run at 0x7fb14e663430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7fb0dd5402c0>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-266375' coro=<_command() done, defined at /usr/local/python...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b8569250>
          │            │        │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '!stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │            │        │    └ GroupMessageEvent(time=1646822080, self_id=507346814, post_type='message', sub_type='normal', user_id=2095652005, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7fb148d49670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-266375' coro=<_command() done, defined at /usr/local/python...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b8569250>
          │       │   │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '!stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │       │   │    └ GroupMessageEvent(time=1646822080, self_id=507346814, post_type='message', sub_type='normal', user_id=2095652005, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │       └ <function Matcher.run at 0x7fb14af48b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-266375' coro=<_command() done, defined at /usr/local/python...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b8569250>
          │    │          │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '!stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │    │          │    └ GroupMessageEvent(time=1646822080, self_id=507346814, post_type='message', sub_type='normal', user_id=2095652005, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │    └ <function Matcher.simple_run at 0x7fb14af48af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'bot': <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>, 'event': GroupMessageEvent(time=1646822080, self_id=5...
                 │    └ <function _ at 0x7fb12f669dc0>
                 └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 490, in _
    await stock.finish(
          │     └ <classmethod object at 0x7fb14af40a90>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 464, in finish
    await cls.send(message, **kwargs)
          │   │    │          └ {'at_sender': True}
          │   │    └ '\n🧾 Successfully sold 1 shares earning 321.17 coins.\nYou have 25 shares left now.\n🟩 Current stock: 321.17 coins,  12.74%'
          │   └ <classmethod object at 0x7fb14af40a30>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 449, in send
    return await bot.send(event=event, message=_message, **kwargs)
                 │   │          │              │           └ {'at_sender': True}
                 │   │          │              └ '\n🧾 Successfully sold 1 shares earning 321.17 coins.\nYou have 25 shares left now.\n🟩 Current stock: 321.17 coins,  12.74%'
                 │   │          └ GroupMessageEvent(time=1646822080, self_id=507346814, post_type='message', sub_type='normal', user_id=2095652005, message_typ...
                 │   └ <function Bot.send at 0x7fb148d49b80>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 247, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
                 │    │                      │     │      │          └ {'at_sender': True}
                 │    │                      │     │      └ '\n🧾 Successfully sold 1 shares earning 321.17 coins.\nYou have 25 shares left now.\n🟩 Current stock: 321.17 coins,  12.74%'
                 │    │                      │     └ GroupMessageEvent(time=1646822080, self_id=507346814, post_type='message', sub_type='normal', user_id=2095652005, message_typ...
                 │    │                      └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                 │    └ <attribute '__class__' of 'object' objects>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 175, in send
    return await bot.send_msg(**params)
                 │              └ {'user_id': 2095652005, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '20...
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 208, in call_api
    return await super().call_api(api, **data)
                                  │      └ {'user_id': 2095652005, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '20...
                                  └ 'send_msg'
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 117, in call_api
    raise exception
          └ <ActionFailed data=None, echo={'seq': 38}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 94, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'user_id': 2095652005, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '20...
                   │    │       │         │     └ 'send_msg'
                   │    │       │         └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                   │    │       └ <function Adapter._call_api at 0x7fb148d49f70>
                   │    └ <nonebot.adapters.onebot.v11.adapter.Adapter object at 0x7fb148e378b0>
                   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/adapter.py", line 107, in _call_api
    return _handle_api_result(
           └ <function _handle_api_result at 0x7fb148ddb5e0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/utils.py", line 81, in _handle_api_result
    raise ActionFailed(**result)
          │              └ {'data': None, 'echo': {'seq': 38}, 'msg': 'SEND_MSG_API_ERROR', 'retcode': 100, 'status': 'failed', 'wording': '请参考 go-cqhtt...
          └ <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>

nonebot.adapters.onebot.v11.exception.ActionFailed: <ActionFailed data=None, echo={'seq': 38}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
[36mATRI[0m | 03-09 18:35:04 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7fb151ddc040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7fb1526624f0>
         │    └ <function MultiCommand.invoke at 0x7fb151de00d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7fb159f4f370>
           │               │       │       └ <function Command.invoke at 0x7fb151ddcaf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7fb159f4f370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7fb148d079d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7fb159f4f370>
           │   │      │    └ <function run at 0x7fb148e328b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7fb151dd88b0>
           └ <click.core.Context object at 0x7fb159f4f370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7fb148e21670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7fb14af0b940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7fb14af0b4c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7fb148c2e670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7fb148c2ea60>
    └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7fb148c2eaf0>
           │       │   └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
           │       └ <function run at 0x7fb14e663430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7fb0dd5402c0>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-268368' coro=<_command() done, defined at /usr/local/python...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b8568430>
          │            │        │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │            │        │    └ GroupMessageEvent(time=1646822098, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7fb148d49670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-268368' coro=<_command() done, defined at /usr/local/python...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b8568430>
          │       │   │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │       │   │    └ GroupMessageEvent(time=1646822098, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │       └ <function Matcher.run at 0x7fb14af48b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-268368' coro=<_command() done, defined at /usr/local/python...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b8568430>
          │    │          │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │    │          │    └ GroupMessageEvent(time=1646822098, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │    └ <function Matcher.simple_run at 0x7fb14af48af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'bot': <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>, 'event': GroupMessageEvent(time=1646822098, self_id=5...
                 │    └ <function _ at 0x7fb12f669dc0>
                 └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 490, in _
    await stock.finish(
          │     └ <classmethod object at 0x7fb14af40a90>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 464, in finish
    await cls.send(message, **kwargs)
          │   │    │          └ {'at_sender': True}
          │   │    └ '\n🧾 Successfully sold 1 shares earning 321.17 coins.\nYou have 29 shares left now.\n🟩 Current stock: 321.17 coins,  12.74%'
          │   └ <classmethod object at 0x7fb14af40a30>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 449, in send
    return await bot.send(event=event, message=_message, **kwargs)
                 │   │          │              │           └ {'at_sender': True}
                 │   │          │              └ '\n🧾 Successfully sold 1 shares earning 321.17 coins.\nYou have 29 shares left now.\n🟩 Current stock: 321.17 coins,  12.74%'
                 │   │          └ GroupMessageEvent(time=1646822098, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
                 │   └ <function Bot.send at 0x7fb148d49b80>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 247, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
                 │    │                      │     │      │          └ {'at_sender': True}
                 │    │                      │     │      └ '\n🧾 Successfully sold 1 shares earning 321.17 coins.\nYou have 29 shares left now.\n🟩 Current stock: 321.17 coins,  12.74%'
                 │    │                      │     └ GroupMessageEvent(time=1646822098, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
                 │    │                      └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                 │    └ <attribute '__class__' of 'object' objects>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 175, in send
    return await bot.send_msg(**params)
                 │              └ {'user_id': 2560359315, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 208, in call_api
    return await super().call_api(api, **data)
                                  │      └ {'user_id': 2560359315, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                                  └ 'send_msg'
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 117, in call_api
    raise exception
          └ <ActionFailed data=None, echo={'seq': 41}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 94, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'user_id': 2560359315, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                   │    │       │         │     └ 'send_msg'
                   │    │       │         └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                   │    │       └ <function Adapter._call_api at 0x7fb148d49f70>
                   │    └ <nonebot.adapters.onebot.v11.adapter.Adapter object at 0x7fb148e378b0>
                   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/adapter.py", line 107, in _call_api
    return _handle_api_result(
           └ <function _handle_api_result at 0x7fb148ddb5e0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/utils.py", line 81, in _handle_api_result
    raise ActionFailed(**result)
          │              └ {'data': None, 'echo': {'seq': 41}, 'msg': 'SEND_MSG_API_ERROR', 'retcode': 100, 'status': 'failed', 'wording': '请参考 go-cqhtt...
          └ <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>

nonebot.adapters.onebot.v11.exception.ActionFailed: <ActionFailed data=None, echo={'seq': 41}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
[36mATRI[0m | 03-09 18:35:18 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7fb151ddc040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7fb1526624f0>
         │    └ <function MultiCommand.invoke at 0x7fb151de00d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7fb159f4f370>
           │               │       │       └ <function Command.invoke at 0x7fb151ddcaf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7fb159f4f370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7fb148d079d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7fb159f4f370>
           │   │      │    └ <function run at 0x7fb148e328b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7fb151dd88b0>
           └ <click.core.Context object at 0x7fb159f4f370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7fb148e21670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7fb14af0b940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7fb14af0b4c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7fb148c2e670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7fb148c2ea60>
    └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7fb148c2eaf0>
           │       │   └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
           │       └ <function run at 0x7fb14e663430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7fb0dd5402c0>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-269594' coro=<_command() done, defined at /usr/local/python...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b854bd90>
          │            │        │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │            │        │    └ GroupMessageEvent(time=1646822113, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7fb148d49670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-269594' coro=<_command() done, defined at /usr/local/python...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b854bd90>
          │       │   │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │       │   │    └ GroupMessageEvent(time=1646822113, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │       └ <function Matcher.run at 0x7fb14af48b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-269594' coro=<_command() done, defined at /usr/local/python...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b854bd90>
          │    │          │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │    │          │    └ GroupMessageEvent(time=1646822113, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │    └ <function Matcher.simple_run at 0x7fb14af48af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'bot': <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>, 'event': GroupMessageEvent(time=1646822113, self_id=5...
                 │    └ <function _ at 0x7fb12f669dc0>
                 └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 490, in _
    await stock.finish(
          │     └ <classmethod object at 0x7fb14af40a90>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 464, in finish
    await cls.send(message, **kwargs)
          │   │    │          └ {'at_sender': True}
          │   │    └ '\n🧾 Successfully sold 1 shares earning 314.25 coins.\nYou have 28 shares left now.\n🟩 Current stock: 314.25 coins,  10.31%'
          │   └ <classmethod object at 0x7fb14af40a30>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 449, in send
    return await bot.send(event=event, message=_message, **kwargs)
                 │   │          │              │           └ {'at_sender': True}
                 │   │          │              └ '\n🧾 Successfully sold 1 shares earning 314.25 coins.\nYou have 28 shares left now.\n🟩 Current stock: 314.25 coins,  10.31%'
                 │   │          └ GroupMessageEvent(time=1646822113, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
                 │   └ <function Bot.send at 0x7fb148d49b80>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 247, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
                 │    │                      │     │      │          └ {'at_sender': True}
                 │    │                      │     │      └ '\n🧾 Successfully sold 1 shares earning 314.25 coins.\nYou have 28 shares left now.\n🟩 Current stock: 314.25 coins,  10.31%'
                 │    │                      │     └ GroupMessageEvent(time=1646822113, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
                 │    │                      └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                 │    └ <attribute '__class__' of 'object' objects>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 175, in send
    return await bot.send_msg(**params)
                 │              └ {'user_id': 2560359315, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 208, in call_api
    return await super().call_api(api, **data)
                                  │      └ {'user_id': 2560359315, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                                  └ 'send_msg'
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 117, in call_api
    raise exception
          └ <ActionFailed data=None, echo={'seq': 44}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 94, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'user_id': 2560359315, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                   │    │       │         │     └ 'send_msg'
                   │    │       │         └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                   │    │       └ <function Adapter._call_api at 0x7fb148d49f70>
                   │    └ <nonebot.adapters.onebot.v11.adapter.Adapter object at 0x7fb148e378b0>
                   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/adapter.py", line 107, in _call_api
    return _handle_api_result(
           └ <function _handle_api_result at 0x7fb148ddb5e0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/utils.py", line 81, in _handle_api_result
    raise ActionFailed(**result)
          │              └ {'data': None, 'echo': {'seq': 44}, 'msg': 'SEND_MSG_API_ERROR', 'retcode': 100, 'status': 'failed', 'wording': '请参考 go-cqhtt...
          └ <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>

nonebot.adapters.onebot.v11.exception.ActionFailed: <ActionFailed data=None, echo={'seq': 44}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
[36mATRI[0m | 03-10 09:46:45 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7fb151ddc040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7fb1526624f0>
         │    └ <function MultiCommand.invoke at 0x7fb151de00d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7fb159f4f370>
           │               │       │       └ <function Command.invoke at 0x7fb151ddcaf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7fb159f4f370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7fb148d079d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7fb159f4f370>
           │   │      │    └ <function run at 0x7fb148e328b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7fb151dd88b0>
           └ <click.core.Context object at 0x7fb159f4f370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7fb148e21670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7fb14af0b940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7fb14af0b4c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7fb148c2e670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7fb148c2ea60>
    └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7fb148c2eaf0>
           │       │   └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
           │       └ <function run at 0x7fb14e663430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7fb0dd5402c0>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-2348595' coro=<_command() done, defined at /usr/local/pytho...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0dc208070>
          │            │        │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '!stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │            │        │    └ GroupMessageEvent(time=1646876800, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7fb148d49670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-2348595' coro=<_command() done, defined at /usr/local/pytho...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0dc208070>
          │       │   │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '!stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │       │   │    └ GroupMessageEvent(time=1646876800, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │       └ <function Matcher.run at 0x7fb14af48b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-2348595' coro=<_command() done, defined at /usr/local/pytho...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0dc208070>
          │    │          │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '!stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │    │          │    └ GroupMessageEvent(time=1646876800, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │    └ <function Matcher.simple_run at 0x7fb14af48af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'bot': <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>, 'event': GroupMessageEvent(time=1646876800, self_id=5...
                 │    └ <function _ at 0x7fb12f669dc0>
                 └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 490, in _
    await stock.finish(
          │     └ <classmethod object at 0x7fb14af40a90>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 464, in finish
    await cls.send(message, **kwargs)
          │   │    │          └ {'at_sender': True}
          │   │    └ '\n🧾 Successfully sold 1 shares earning 429.47 coins.\nYou have 24 shares left now.\n🟩 Current stock: 429.47 coins,  62.99%'
          │   └ <classmethod object at 0x7fb14af40a30>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 449, in send
    return await bot.send(event=event, message=_message, **kwargs)
                 │   │          │              │           └ {'at_sender': True}
                 │   │          │              └ '\n🧾 Successfully sold 1 shares earning 429.47 coins.\nYou have 24 shares left now.\n🟩 Current stock: 429.47 coins,  62.99%'
                 │   │          └ GroupMessageEvent(time=1646876800, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
                 │   └ <function Bot.send at 0x7fb148d49b80>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 247, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
                 │    │                      │     │      │          └ {'at_sender': True}
                 │    │                      │     │      └ '\n🧾 Successfully sold 1 shares earning 429.47 coins.\nYou have 24 shares left now.\n🟩 Current stock: 429.47 coins,  62.99%'
                 │    │                      │     └ GroupMessageEvent(time=1646876800, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
                 │    │                      └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                 │    └ <attribute '__class__' of 'object' objects>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 175, in send
    return await bot.send_msg(**params)
                 │              └ {'user_id': 2560359315, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 208, in call_api
    return await super().call_api(api, **data)
                                  │      └ {'user_id': 2560359315, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                                  └ 'send_msg'
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 117, in call_api
    raise exception
          └ <ActionFailed data=None, echo={'seq': 351}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 94, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'user_id': 2560359315, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                   │    │       │         │     └ 'send_msg'
                   │    │       │         └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                   │    │       └ <function Adapter._call_api at 0x7fb148d49f70>
                   │    └ <nonebot.adapters.onebot.v11.adapter.Adapter object at 0x7fb148e378b0>
                   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/adapter.py", line 107, in _call_api
    return _handle_api_result(
           └ <function _handle_api_result at 0x7fb148ddb5e0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/utils.py", line 81, in _handle_api_result
    raise ActionFailed(**result)
          │              └ {'data': None, 'echo': {'seq': 351}, 'msg': 'SEND_MSG_API_ERROR', 'retcode': 100, 'status': 'failed', 'wording': '请参考 go-cqht...
          └ <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>

nonebot.adapters.onebot.v11.exception.ActionFailed: <ActionFailed data=None, echo={'seq': 351}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
[36mATRI[0m | 03-10 10:20:17 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7fb151ddc040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7fb1526624f0>
         │    └ <function MultiCommand.invoke at 0x7fb151de00d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7fb159f4f370>
           │               │       │       └ <function Command.invoke at 0x7fb151ddcaf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7fb159f4f370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7fb148d079d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7fb159f4f370>
           │   │      │    └ <function run at 0x7fb148e328b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7fb151dd88b0>
           └ <click.core.Context object at 0x7fb159f4f370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7fb148e21670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7fb14af0b940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7fb14af0b4c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7fb148c2e670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7fb148c2ea60>
    └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7fb148c2eaf0>
           │       │   └ <uvicorn.server.Server object at 0x7fb0dd5935b0>
           │       └ <function run at 0x7fb14e663430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7fb0dd5402c0>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-2417582' coro=<_command() done, defined at /usr/local/pytho...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b9924dc0>
          │            │        │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '！stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │            │        │    └ GroupMessageEvent(time=1646878812, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7fb148d49670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-2417582' coro=<_command() done, defined at /usr/local/pytho...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b9924dc0>
          │       │   │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '！stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │       │   │    └ GroupMessageEvent(time=1646878812, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │       └ <function Matcher.run at 0x7fb14af48b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7fb14aecc160>: <Task finished name='Task-2417582' coro=<_command() done, defined at /usr/local/pytho...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7fb0b9924dc0>
          │    │          │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '！stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │    │          │    └ GroupMessageEvent(time=1646878812, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
          │    └ <function Matcher.simple_run at 0x7fb14af48af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'bot': <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>, 'event': GroupMessageEvent(time=1646878812, self_id=5...
                 │    └ <function _ at 0x7fb12f669dc0>
                 └ <Dependent call=<function _ at 0x7fb12f669dc0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 490, in _
    await stock.finish(
          │     └ <classmethod object at 0x7fb14af40a90>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 464, in finish
    await cls.send(message, **kwargs)
          │   │    │          └ {'at_sender': True}
          │   │    └ '\n🧾 Successfully sold 3 shares earning 1064.67 coins.\nYou have 4 shares left now.\n🟩 Current stock: 354.89 coins,  34.68%'
          │   └ <classmethod object at 0x7fb14af40a30>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 449, in send
    return await bot.send(event=event, message=_message, **kwargs)
                 │   │          │              │           └ {'at_sender': True}
                 │   │          │              └ '\n🧾 Successfully sold 3 shares earning 1064.67 coins.\nYou have 4 shares left now.\n🟩 Current stock: 354.89 coins,  34.68%'
                 │   │          └ GroupMessageEvent(time=1646878812, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
                 │   └ <function Bot.send at 0x7fb148d49b80>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 247, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
                 │    │                      │     │      │          └ {'at_sender': True}
                 │    │                      │     │      └ '\n🧾 Successfully sold 3 shares earning 1064.67 coins.\nYou have 4 shares left now.\n🟩 Current stock: 354.89 coins,  34.68%'
                 │    │                      │     └ GroupMessageEvent(time=1646878812, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
                 │    │                      └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                 │    └ <attribute '__class__' of 'object' objects>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 175, in send
    return await bot.send_msg(**params)
                 │              └ {'user_id': 1834580024, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '18...
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 208, in call_api
    return await super().call_api(api, **data)
                                  │      └ {'user_id': 1834580024, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '18...
                                  └ 'send_msg'
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 117, in call_api
    raise exception
          └ <ActionFailed data=None, echo={'seq': 362}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 94, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'user_id': 1834580024, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '18...
                   │    │       │         │     └ 'send_msg'
                   │    │       │         └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
                   │    │       └ <function Adapter._call_api at 0x7fb148d49f70>
                   │    └ <nonebot.adapters.onebot.v11.adapter.Adapter object at 0x7fb148e378b0>
                   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7fb0dcae7b20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/adapter.py", line 107, in _call_api
    return _handle_api_result(
           └ <function _handle_api_result at 0x7fb148ddb5e0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/utils.py", line 81, in _handle_api_result
    raise ActionFailed(**result)
          │              └ {'data': None, 'echo': {'seq': 362}, 'msg': 'SEND_MSG_API_ERROR', 'retcode': 100, 'status': 'failed', 'wording': '请参考 go-cqht...
          └ <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>

nonebot.adapters.onebot.v11.exception.ActionFailed: <ActionFailed data=None, echo={'seq': 362}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
